safari/tickety-tick.xcodeproj/project.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+	<string>
+// ___FILENAME___
+//</string>
```

Comment:
~weird~ interesting syntax 👀

Reply:
Yeah. I tried setting it to an empty string to remove the whole header (because I don't really see the point in those). But then XCode still inserts an empty line comment in each new file. It's a known issue. 🙄